flickr.py

```diff
@@ -0,0 +1,48 @@
+import requests
+from logging import logging
+import os
+
+key = os.environ.get('flickr_key')
+
+def getImage(yelp_dish):
+# Docs at https://www.flickr.com/services/api/explore/flickr.photos.search
+# Search for pictures, modify to search for whatever tag you want
+flickerSearchURL = 'https://api.flickr.com/services/rest/'
+params = {
+'method': 'flickr.photos.search',
+'api_key': key,
+'text': yelp_dish,
+'format': 'json',
+'nojsoncallback': '1',
+'sort': 'relevance',
+}
+# Search flickr for pictures
+flickrResponse = requests.get(flickerSearchURL, params=params)
+# get json back
+flickrResponseJson = flickrResponse.json()
+
+# Get first json object ('photos') which contains another json object ('photo') which is an json array; each
+# element represents one photo. Take element 1
+firstResponsePhoto = flickrResponseJson['photos']['photo'][0]
+#confirm to get the exact json
+print(firstResponsePhoto)
```

Comment:
Is this print for debugging?

Reply:
it is used to confirm whether we are getting the exact photo

Reply:
@xm6264jz gotcha. We might want to omit it in prod and use logging instead

Reply:
resolving convo